android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -1255,11 +1250,12 @@ public void setSrc(
             boolean isOriginalSourceNull = srcUrl == null;
             boolean isSourceEqual = url.equals(srcUrl);
 
+            this.isImaDaiStreamLoaded = false;
             if (ima != null && !ima.isEmpty()) {
-                this.isImaStream = true;
-                this.imaSrc = new RNImaSource(ima);
+                this.isImaDaiStream = true;
+                this.imaDaiSrc = new RNImaDaiSource(ima);
             } else {
-                this.isImaStream = false;
+                this.isImaDaiStream = false;
```

Comment:
Can we use the same logic here as we're using on the mobile video player library to keep the libraries as consistent as possible?

```java
            this.isImaDaiStream = false;
            this.isImaDaiStreamLoaded = false;
            if (ima != null && !ima.isEmpty()) {
                this.isImaDaiStream = true;
                this.imaDaiSrc = new RNImaDaiSource(ima);
            } 
```